packages/front-end/components/Features/CodeSnippetModal.tsx

```diff
@@ -87,7 +87,7 @@ export default function CodeSnippetModal({
   const [installationOpen, setInstallationOpen] = useState(true);
   const [setupOpen, setSetupOpen] = useState(true);
   const [usageOpen, setUsageOpen] = useState(true);
-  const [attributesOpen, setAttributesOpen] = useState(false);
+  const [attributesOpen, setAttributesOpen] = useState(true);
```

Comment:
FYI I toggled this open by default because I asked where it was but didn't realize it was on the page. I think it should be open for readability/scannability but if people disagree I can revert this change.